easybuild/easyconfigs/r/ROOT/ROOT-v5.34.26-intel-2015a.eb

```diff
@@ -0,0 +1,56 @@
+name = 'ROOT'
+version = 'v5.34.26'
+
+homepage = 'http://root.cern.ch/drupal/'
+description = """The ROOT system provides a set of OO frameworks with all the functionality
+    needed to handle and analyze large amounts of data in a very efficient way."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+sources = ['%(namelower)s_%(version)s.source.tar.gz']
+source_urls = ['ftp://root.cern.ch/root/']
+patches = [
+    'configure_FftwFromMkl_28.patch',
+    'ROOT-v5_recent-ifort.patch',
+]
+
+python = 'Python'
+pyver = '2.7.9'
+
+dependencies = [
+    ('GSL', '1.16'),
+    ('Mesa', '10.4.5', '-%s-%s' % (python, pyver)),
+    ('libxml2', '2.9.2', '-%s-%s' % (python, pyver)),
+    ('PCRE', '8.36'),
+    ('CFITSIO', '3.37'),
+    ('freetype', '2.5.5'),
+    (python, pyver),
+    ('zlib', '1.2.8'),
+    ('libXft', '2.3.2', 'libX11-1.6.3-%s-%s' % ((python, pyver)),
```

Comment:
@hajgato: you have an extra `(` here, before the `python`, so 3x `(`, only 2x `)` => syntax error